spec/defines/admin_spec.rb

```diff
@@ -5,14 +5,14 @@
     context "on #{os}" do
       let(:facts) { os_facts }
       let(:title) { 'help' }
-      let(:pre_condition) do
-        <<-PUPPET
-          include pulpcore
-        PUPPET
-      end
 
-      context 'with a fixed pulp_settings' do
-        let(:params) { { pulp_settings: '/etc/pulpcore/settings.py' } }
+      context 'with a fixed pulp_settings and static_root' do
+        let(:params) do
+          super().merge(
+            pulp_settings: '/etc/pulpcore/settings.py',
+            static_root: '/var/lib/pulp/assets'
```

Comment:
Generally I'm fan of trailing comma's. The benefit is that if you add another line, you don't need to modify this line. You can also sort them. Same goes for the line you added near the end.